packages/@o3r/rules-engine/src/components/rules-engine/rule-key-value/rule-key-value-pres.template.html

```diff
@@ -3,12 +3,22 @@
 <ng-container *ngIf="oldValue">
   <pre class="input-value"
     [class.limit-characters]="shouldLimitCharactersForOldValue"
-    (click)="shouldLimitCharactersForOldValue = !shouldLimitCharactersForOldValue">{{isOldValuePrimitiveType ? oldValue : (oldValue | json)}}</pre>
+    (click)="shouldLimitCharactersForOldValue = !shouldLimitCharactersForOldValue"
+    (keyup.enter)="shouldLimitCharactersForOldValue = !shouldLimitCharactersForOldValue"
+    tabindex="0"
+  >
+    {{isOldValuePrimitiveType ? oldValue : (oldValue | json)}}
+  </pre>
   <button (click)="copyToClipBoard(oldValue)" *ngIf="isClipBoardFeatureAvailableForOldValue" title="Copy to clipboard">📋</button>
   →
 </ng-container>
 <pre class="input-value"
```

Comment:
shouldn't it be a button in that case? We can have a button without actual styling and get it similar visually

Reply:
It should not be a button it is only a click to change the css class to have the ellipsis or not 
I don't want the screen reader to be aware of that because the innerContent will be the same and does not explain the action that a button will made 
for me in this case we want only the click and the keyboard navigation but not a button